src/components/stack-chart/Settings.js

```diff
@@ -11,28 +11,34 @@ import {
   changeInvertCallstack,
```

Comment:
A lot in this file is being copied from `ProfileCallTreeSettings.js` but I don't have a good idea to factorize them yet.